app/src/main/res/layout/movie_details_custom_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
👍 

Btw, if you want to be able to preview this layout in the Android Studio designer then you can use `tools:parentLayout=FrameLayout` and ` tools:layout_width=match_parent` and all the rest of the android attributes just with the `tools` prefix.